src/main/java/org/intellij/sdk/codesync/HandleBuffer.java

```diff
@@ -127,6 +128,10 @@ public static void handleBuffer() {
                 if (isSuccess) {
                     System.out.printf("Diff file '%s' successfully processed.\n", diffFile.originalDiffFile.getPath());
                     diffFile.delete();
+
+                    // We also need to disconnect existing connections here,
+                    // otherwise the server cache causes an error that user is not able to edit new file.
```

Comment:
What do you mean by `that user is not able to edit new file`?

Reply:
User is able to edit the the file but those updates error out and are not sent to the server until pyCharm us restarted. I should reword the comment to avoid confusion.

Reply:
Oh, seems like IDE issue

Reply:
Thank you for the clarification